docs/tutorial/get-started.md

```diff
@@ -44,300 +46,329 @@ This chapter will show you how to deploy using Helm and run Vald on your kuberne
+   ```
+
+1. Deploy Vald using helm
+
+   ```bash
+   helm repo add vald https://vald.vdaas.org/charts
+   helm install vald vald/vald --values example/helm/values-scylla.yaml
+   ```
+
+1. Verify
+
+   ```bash
+   kubectl get pods
+   ```
+
+   <details><summary>Example output</summary><br>
+   If the deployment is successful, all Vald components should be running.
+
+   ```bash
+   NAME                                       READY   STATUS      RESTARTS   AGE
+   scylla-init-vhdp5                          0/1     Completed   0          7m12s
+   vald-agent-ngt-0                           1/1     Running     0          7m12s
+   vald-agent-ngt-1                           1/1     Running     0          7m12s
+   vald-agent-ngt-2                           1/1     Running     0          7m12s
+   vald-agent-ngt-3                           1/1     Running     0          7m12s
+   vald-agent-ngt-4                           1/1     Running     0          7m12s
+   vald-agent-ngt-5                           1/1     Running     0          7m12s
+   vald-backup-gateway-68c8b4ffd4-df8zp       1/1     Running     0          6m56s
+   vald-backup-gateway-68c8b4ffd4-dmwrd       1/1     Running     0          6m56s
+   vald-backup-gateway-68c8b4ffd4-nm8f7       1/1     Running     0          7m12s
+   vald-discoverer-7f9f697dbb-q44qh           1/1     Running     0          7m11s
+   vald-lb-gateway-6b7b9f6948-4z5md           1/1     Running     0          7m12s
+   vald-lb-gateway-6b7b9f6948-68g94           1/1     Running     0          6m56s
+   vald-lb-gateway-6b7b9f6948-cvspq           1/1     Running     0          6m56s
+   vald-manager-backup-5fb5f8dc7-h22sv        1/1     Running     0          7m12s
+   vald-manager-backup-5fb5f8dc7-ncrw4        1/1     Running     0          6m56s
+   vald-manager-backup-5fb5f8dc7-nzbkh        1/1     Running     0          6m56s
+   vald-manager-compressor-78bf64459f-27ckg   1/1     Running     0          6m56s
+   vald-manager-compressor-78bf64459f-9kl9b   1/1     Running     0          7m12s
+   vald-manager-compressor-78bf64459f-dkx24   1/1     Running     0          6m56s
+   vald-manager-index-74c7b5ddd6-jrnlw        1/1     Running     1          7m12s
+   vald-meta-747f757bbb-9v5xz                 1/1     Running     0          7m12s
+   vald-meta-747f757bbb-mpwqp                 1/1     Running     0          6m56s
+   vald-meta-gateway-8c5f55dd-8fsch           1/1     Running     0          6m56s
+   vald-meta-gateway-8c5f55dd-sdd5q           1/1     Running     0          7m12s
+   vald-meta-gateway-8c5f55dd-vfkn6           1/1     Running     0          6m56s
+   ```
+
+   </details>
 
 ### Run using example code
 
 This chapter shows how to perform a search action in Vald with fashion-mnist dataset.
 
 1. Port Forward
 
-    ```bash
-    kubectl port-forward deployment/vald-gateway 8081:8081
-    ```
+   ```bash
+   kubectl port-forward deployment/vald-lb-gateway 8081:8081
```

Comment:
```suggestion
   kubectl port-forward deployment/vald-meta-gateway 8081:8081
```

Reply:
oh....ok, thx.
I'll confirm it soon, too.